tools/roslaunch/src/roslaunch/substitution_args.py

```diff
@@ -47,6 +47,9 @@
 import rosgraph.names
 import rospkg
 
+from . import cached
```

Comment:
I am not sure why the import was being done below but there might be a good reason for it.Therefore please keep the import in the same location as before.